pcdsdevices/widgets/qmini.py

```diff
@@ -166,36 +170,42 @@ def fix_plot_domain(self):
         does not work as expect. Fix it when the device signals connect.
         """
         plot = self.ui.plot
-        x_min = int(self.device.wavelengths.get().min())
-        x_max = int(self.device.wavelengths.get().max())
-
-        if not x_max > x_min:
-            # Restart the timer if these signals are still None
-            self._plot_timer.start()
-
+        # Avoid infinite window spawning with numpy fails on subscription
+        try:
+            x_min = int(self.device.wavelengths.get().min())
+            x_max = int(self.device.wavelengths.get().max())
+        except Exception as e:
+            logger.exception(e)
```

Comment:
If this errors out, we won't have values for x_min/x_max, making the rest of this method moot.  

We probably just return here?

Same comment goes for the below `auto_range_y`

Reply:
Sorry I missed the else.  This can work, but it's worth considering aborting early anyway